xcube/cli/versions.py

```diff
@@ -0,0 +1,70 @@
+# The MIT License (MIT)
+# Copyright (c) 2019 by the xcube development team and contributors
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
```

Comment:
This line is a little bit too long. In xcube.util.versions I have adjusted it so it corresponds to our guidelines. We should use that whenever we create new modules.

Reply:
Same for _all_ xcube modules. Let's fix this in one go later.